web/packages/teleport/src/config.ts

```diff
@@ -40,6 +40,7 @@ const cfg = {
   isDashboard: false,
   tunnelPublicAddress: '',
   recoveryCodesEnabled: false,
+  isUsageBasedBilling: false,
```

Comment:
can we copy over the comment describing this flag? (we are working on adding more helpful comments in the web UI)